File: src/Modelos/Usuario.js
export default class Usuario {

    constructor(usuario, senha) {
        this._usuario = usuario;
        this._senha = senha;
    }

    get usuario() {
        return this._usuario;
    };

    set usuario(usuario) {
        return this._usuario = usuario;
    }

    get senha() {
        return this._senha;
    }

    set senha(senha) {
        return this._senha = senha;
    }
    
}

File: src/main/java/projetoSoftware/com/mercado/modelo/Usuario.java
package projetoSoftware.com.mercado.modelo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "usuario")
public class Usuario {

    @Id
    @Column(name = "identificador")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int identificador;
    @Column(name = "usuario")
    private String usuario;
    @Column(name = "senha")
    private String senha;
    @Column(name = "nome")
    private String nome;
    @Column(name = "cargo")
    private String cargo;

}